configs/piksiv3_defconfig

```diff
@@ -1,13 +1,20 @@
+BR2_GLOBAL_PATCH_DIR="$(BR2_EXTERNAL_piksi_buildroot_PATH)/board/piksiv3/patches"
 BR2_arm=y
 BR2_cortex_a9=y
 BR2_ARM_ENABLE_NEON=y
 BR2_ARM_ENABLE_VFP=y
 BR2_ARM_FPU_NEON=y
-BR2_JLEVEL=0
-BR2_GLOBAL_PATCH_DIR="$(BR2_EXTERNAL_piksi_buildroot_PATH)/board/piksiv3/patches"
 BR2_TOOLCHAIN_EXTERNAL=y
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM=y
+BR2_TOOLCHAIN_EXTERNAL_DOWNLOAD=y
+BR2_TOOLCHAIN_EXTERNAL_URL="http://toolchains.free-electrons.com/downloads/releases/toolchains/armv7-eabihf/tarballs/armv7-eabihf--glibc--stable-2018.02-2.tar.bz2"
+BR2_TOOLCHAIN_EXTERNAL_GCC_6=y
+BR2_TOOLCHAIN_EXTERNAL_HEADERS_4_1=y
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
+BR2_TOOLCHAIN_EXTERNAL_CXX=y
+BR2_TOOLCHAIN_EXTERNAL_GDB_SERVER_COPY=y
+BR2_TOOLCHAIN_GLIBC_GCONV_LIBS_COPY=y
 BR2_TARGET_OPTIMIZATION="-flto"
-BR2_TARGET_CFLAGS="-flto"
 BR2_TARGET_LDFLAGS="-flto"
```

Comment:
Is this redundant to the line in lto.mk that also sets -flto on BR2_TARGET_LDFLAGS?

Reply:
Yes, it should be

Reply:
I haven't nailed down all the interactions with the flags/variables in buildroot, but it seems the CFLAGS one isn't used (or isn't recognized) by the buildroot config system, so when `make menuconfig` is run or buildroot it drops the `BR2_TARGET_CFLAGS` variable.